lib/Zim.js

```diff
@@ -99,6 +100,24 @@ Zim.prototype.computeZimName = function() {
     return (this.publisher ? this.publisher.toLowerCase() + '.' : '') + this.env.computeFilenameRadical(false, true, true);
 };
 
+Zim.prototype.computeZimTags = function() {
+    var tags = [];
+
+    /* Mediawiki hostname radical */
+    var mwUrlHostParts = urlParser.parse(this.env.mw.base).host.split(".");
+    var mwUrlHostPartsTag = mwUrlHostParts.length > 1 ?
+	mwUrlHostParts[mwUrlHostParts.length - 2] : mwUrlHostParts[mwUrlHostParts.length - 1]
+    tags.push(mwUrlHostPartsTag.toLowerCase());
+
+    /* novid/nopic */
+    this.env.nopic ? tags.push('nopic') : this.env.novid ? tags.push('novid') : undefined;
```

Comment:
I don't like using ternary operators for this ... Better to use the same if (this.env...) pattern you use below on lines 115 and 116.

Reply:
Agree